install/kubernetes/cilium/values.yaml

```
@@ -106,6 +106,10 @@ global:
     enabled: false
     port: 6942
 
+  # blacklistConflictingRoutes instructs the cilium-agent to blacklist or not
+  # local routes.
```

nit, I think it would be good to mirror the exact wording of the daemon option just to make sure there's no ambiguity between different ways that this option can be configured:

```suggestion
  # blacklistConflictingRoutes instructs the cilium-agent whether to blacklist 
  # IP allocations conflicting with local non-cilium routes.
```

---

You are right, it's fixed in the last commit.